pages/404.module.css.map

```diff
@@ -1 +1 @@
-{"version":3,"sourceRoot":"","sources":["404.module.scss"],"names":[],"mappings":"AAAA;EACE;EACA","file":"404.module.css"}
\ No newline at end of file
+{"version":3,"sourceRoot":"","sources":["../styles/extends.scss","404.module.scss","../styles/mixins.scss","../styles/variables.scss"],"names":[],"mappings":"AAYA;EACE;EACA;EACA;;;ACbF;ECDE,WDGmB;ECFnB,aDEyB;EACzB,OECU;EFAV;EACA","file":"404.module.css"}
```

Comment:
Maps usually have to be generated in development mode, or in rare cases in production to debug something. They should not be added to the repo. Remove it and gitignore

Reply:
Thanks, fixed